src/main/resources/templates/java/test/staticCodeAnalysisConfig/checkstyle-configuration.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
     "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
-    "https://checkstyle.org/dtds/configuration_1_3.dtd">
+    "https://checkstyle.sourceforge.io/dtds/configuration_1_3.dtd">
```

Comment:
```suggestion
    "https://checkstyle.org/dtds/configuration_1_3.dtd">
```
The sourceforge website is only the legacy one. checkstyle.org is the new one.

The published default config.xml uses the checkstyle.org domain (https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-checks.xml), and the documentation only links to checkstyle.sourceforge.io for older release information (https://checkstyle.org/index.html#Previous_Version_Documentation).

The URL structure of both pages is the same, the fix should be doable by search-and-replace.

Reply:
Thanks for pointing this out. The old links did not work and Google always pointed to the sourgeforce one, so I used them here.